package.json

```diff
@@ -20,7 +20,7 @@
     "url": "https://github.com/osartun/Backbone.Undo.js/issues"
   },
   "dependencies": {
-    "backbone": "~1.0.0",
-    "underscore": "~1.4.4"
+    "backbone": "1.0.0 - 1.2.1",
+    "underscore": "1.4.4 - 1.8.3"
```

Comment:
Thanks for testing this with all these versions!
However, if the versions are determined like this, this version specification needs to be updated when new versions of backbone or underscore are released, right? Because then it would be better to determine the version number in a way that doesn't require this file to be updated, like `">=1.0.0"` / `">=1.4.4`, right?

I wasn't aware that `~` restricts the versions so much, just [read it here](http://the-phpjs-ldc.rgou.net/npm/doc/misc/semver.md#ranges)